docs/tutorials/template.ipynb

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
<p><code>Open a catalog </code> to avoid word <code>load</code> that can be misinterpreted.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/784/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>